services/openshiftbuilddeploy/src/index.js

```diff
@@ -245,6 +253,10 @@ const messageConsumer = async msg => {
                       {
                         "name": "NATIVE_CRON_POD_MINIMUM_FREQUENCY",
                         "value": NativeCronPodMinimumFrequency
+                      },
+                      {
+                        "name": "AVAILABILITY",
```

Comment:
you inject `AVAILABILITY` into the build pod, but I never see this used?

Reply:
ah yeah, I was originally having the availability injected and then figuring out the HA/SA contact in the build-deploy-docker-compose scripts, but switched tacts